citest/json_contract/value_observation_verifier.py

```diff
@@ -80,22 +80,25 @@ def add_constraint(self, constraint):
 
   def contains_path_match(self, path, match_spec, min=1, max=None,
                           **kwargs):
+    match_kwargs = kwargs.pop('match_kwargs', {})
```

Comment:
It's not clear what the options for `match_kwargs` are without drilling into the predicates themselves. If the options are the same for each type of predicate, then I think a comment or docstring would be helpful for these functions.